doc/source/release/1.16.0-notes.rst

```diff
@@ -17,114 +17,1035 @@ run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
+
+********************
+Expired Deprecations
+********************
+- ``scipy.sparse.conjtransp`` has been removed. Use ``.T.conj()`` instead.
+- The ``quadrature='trapz'`` option has been removed from
+  `scipy.integrate.quad_vec` and ``scipy.stats.trapz`` has been removed. Use
+  ``trapezoid`` in both instances instead.
+- `scipy.special.comb` and `scipy.special.perm` now raise when ``exact=True``
+  and arguments are non-integral.
+- Support for inference of the two sets of measurements from the single
+  argument ``x`` has been removed from `scipy.stats.linregress`. The data
+  must be specified separately as ``x`` and ``y``.
+- Support for NumPy masked arrays has been removed from
+  `scipy.stats.power_divergence` and `scipy.stats.chisquare`.
+
 
 ******************************
 Backwards incompatible changes
 ******************************
+- Several of the `scipy.linalg` functions for solving a linear system (e.g.
+  `scipy.linalg.solve`) documented that the RHS argument must be either 1-D or
+  2-D but did not always raise an error when the RHS argument had more the
+  two dimensions. Now, many-dimensional right hand sides are treated according
+  to the rules specified in :ref:`linalg_batch`.
+- `scipy.stats.bootstrap` now explicitly broadcasts elements of ``data`` to the
+  same shape (ignoring ``axis``) before performing the calculation.
 
 *************
 Other changes
 *************
+- The ``lm`` method of ``least_squares`` function now has a different behavior
+  for the maximum number of function evaluations, ``max_nfev``. The default for
+  the ``lm`` method is changed to ``100 * n``, for both a callable and a
+  numerically estimated jacobian. This limit on function evaluations excludes
+  those used for any numerical estimation of the Jacobian. Previously the
+  default when using an estimated jacobian was ``100 * n * (n + 1)``, because
+  the method included evaluations used in the estimation. In addition, for the
+  ``lm`` method the number of function calls used in Jacobian approximation
+  is no longer included in ``OptimizeResult.nfev``. This brings the behavior
+  of ``lm``, ``trf``, and ``dogbox`` into line.
+- For ``Cython>=3.1.0b1``, SciPy now uses the new
+  ``cython --generate-shared`` functionality, which reduces the total size of
+  SciPy's wheels and on-disk installations significantly.
+- The output of `scipy.stats.wrapcauchy` ``rvs`` is now mapped to the unit circle
+  between 0 and ``2 * pi``.
+- The vendored Qhull library was upgraded from version 2019.1 to 2020.2.
+- A new build option ``-Duse-system-libraries`` has been added. It allows
+  opting in to using system libraries instead of using vendored sources.
+  Currently ``Boost.Math`` and ``Qhull`` are supported as system build
+  dependencies.
+- The ``namedtuple``-like bunch objects returned by some SciPy functions
+  now have improved compatibility with the ``polars`` library.
+- The testsuite thread safety with free-threaded CPython has improved.
```

Comment:
I'll rephrase and add to this. It was more than just the test suite -e.g.,  we finished making `sparse` and `special` thread-safe.